### ISSUE TITLE
What's new screen is top heavy

### ISSUE THREAD

**Issue by danirabbit** [elementary/onboarding#106]
![Screenshot from 2020-07-23 11 47 01](https://user-images.githubusercontent.com/7277719/88325976-6ce36480-ccda-11ea-8291-5d0f72649c50.png)

Not sure what to do about that. Maybe break the blog link out?

**Comment by cassidyjames** [comment]
Yeah, I agree. I forgot to report this. :sweat: I think breaking the blog link out below like we do for the welcome screen makes sense. While we're at it, are there other resources we'd want to link to?

**Comment by dar5hak** [comment]
Would the official Mastodon and Twitter accounts make sense here? But then that would call for a change in the copy, since those aren't exactly "detailed information".

**Comment by zeebok** [comment]
Started to work on this:
![Screenshot from 2020-10-21 23-21-47](https://user-images.githubusercontent.com/1378344/96820591-51304d00-13f4-11eb-9a6c-9255eff20f14.png)
It does feel a little nicer with the empty space filled in, if not a little sparse. Is there any other good resource for eOS information that comes to mind?  Would it make sense to maybe reword this to also include sponsoring on Patreon or GitHub to continue getting new features and improvements?

**Comment by dar5hak** [comment]
@zeebok That's a great start!

We might not need the name though, since the user knows they're on elementary OS, and there's a logo.

I think we could go with something like “Check out our blog…” or “Read our blog…” (with some changes to the message above). This would be better than saying “Blog…”, which would not consume much space. Thoughts?

**Comment by zeebok** [comment]
Good call about the name and logo. Did a quick rework to incorporate that and my thought on linking to patreon:
![Screenshot from 2020-10-22 15-34-30](https://user-images.githubusercontent.com/1378344/96921240-343f5c80-147c-11eb-8b6f-e0dbe987df51.png)
Looking at it a little more, I wonder if "patron" is the wrong word since only people that know about Patreon will really get the concept?

**Comment by cassidyjames** [comment]
@zeebok I think re-using the "Get Involved" link might be better here, actually! It has a section for funding as well, but might come off better. Otherwise, I might add something like a "Fund elementary OS" link that links directly to `https://elementary.io/get-involved#funding`

**Comment by zeebok** [comment]
So something like this?
![Screenshot from 2020-10-22 20-04-54](https://user-images.githubusercontent.com/1378344/96942029-f6edc580-14a1-11eb-9dc3-9b8ba552e45e.png)

**Comment by zeebok** [comment]
I just realized this would be the same as the Welcome View. I should probably just use the same entry for get involved here...

**Comment by rakshit087** [comment]
Looking great!
